frameworks/digital-outcomes-and-specialists/questions/briefs/specialistRole.yml

```diff
@@ -76,6 +81,10 @@ options:
       Develop and deliver an effective user-focused digital service.
       Manage the full product lifecycle, including user research, design, delivery and continuous improvement.
   -
+    label: Technical Architect
+    value: technical_architect
```

Comment:
All the other values here are camelCased - these should be `performanceAnalyst` and `technicalArchitect`.
See: https://github.com/alphagov/digitalmarketplace-frameworks/blob/master/frameworks/digital-outcomes-and-specialists/manifests/edit_submission.yml#L65